migration/migrate_modifications_in_directory_to_composite_modification.sql

```diff
@@ -7,8 +7,11 @@ with
 insert into networkmodifications.composite_modification_sub_modifications(id, modification_id, modifications_order)
 select n.id as id, m.id as modification_id, row_number() over () from modifications_ids m full join new_composite n on true=true;
 delete from directory.element where type = 'MODIFICATION';
+new_directory as (insert into directory.element(id, name, owner, parent_id, type, description, creation_date, last_modification_date, last_modified_by, stash_date, stashed)
+values (gen_random_uuid(), 'Modification_composite_migration', 'admin',
```

Comment:
The script should be as much transparent as possible for users !
Do not create a new create. Modifications should stay in the directory they are